types/react-twitter-embed/index.d.ts

```diff
@@ -0,0 +1,85 @@
+// Type definitions for react-twitter-embed 3.0
+// Project: https://github.com/saurabhnemade/react-twitter-embed#readme
+// Definitions by: Cyrille <https://github.com/zozoens31>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.5
+import * as React from 'react';
+
+export interface BaseProps {
+    onLoad?: (element: HTMLElement) => void;
+    options?: {};
+    placeholder?: string | React.ReactElement;
+}
+
+export namespace timeline {
```

Comment:
Is there a particular reason you need to wrap these types in a namespace?
As far as I can tell, it's not really helping you in any way, and you don't seem to be using it in your test file.